.github/workflows/lint.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
   flake8_py3:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
```

Comment:
This is not directly related to us, but also works :)

Reply:
do we want to pin this?

Reply:
Not sure, it wasn´t before, so I kept it like it was, do you foresee any issues? (if so I'll pin it in a different PR)

Reply:
@david-caro I mean, it is unlikely since we use it without arguments and its probably the most used action from github itself. But in theory a new version could change behaviour and break something in our CI. 

CC @supertassu

Reply:
Makes sense yes, I'll open a PR for it :)